.pre-commit-config.yaml

```diff
@@ -201,6 +201,18 @@ repos:
           - eslint-plugin-promise@6.1.1
           - eslint-plugin-vue@8.7.1
 
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v3.0.3
+    hooks:
+      - id: prettier
+        name: prettier
+        entry: prettier
+        files: ^client/
+        types_or: [ vue, ts, tsx, javascript, jsx, html ]
+        args: [ --fix ]
+        stages:
+          - manual
```

Comment:
That is nice to have added a configuration for `prettier` on `pre-commit`, but it will likely be forgotten (since dependabot will update `package.json` and not `pre-commit-config.yaml`).

I would recommend either:

- Doing something like `ruff` or `mypy` on `pre-commit` (i.e.: running the command `npx prettier . --fix`).
- Handling the version of the hook using `version_udpater` script.